language/diem-framework/tests/move_unit_test.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use diem_vm::natives::diem_natives;
+use move_unit_test::UnitTestingConfig;
+
+move_unit_test::register_move_unit_tests!(
+    UnitTestingConfig::default_with_bound(Some(100_000)),
+    ".",
+    r".*\.move$",
+    r"../move-stdlib/modules",
```

Comment:
Please avoid referencing contents in another crate via paths. This makes it hard for us to see the dependency.

I'd suggest you use `move_stdlib::move_stdlib_modules_full_path()` instead.

Reply:
Correct about genesis. We may want to add this functionality later (custom init functions), but I feel as though adding such a feature would run counter to the philosophy of the testing framework (minimality + transparency of its semantics).

Reply:
Yeah the current solution actually looks pretty neat to me. I especially like the fact that it doesn't require a special mechanism for initialization.